clang/lib/Tooling/DumpTool/ClangSrcLocDump.cpp

```diff
@@ -111,7 +111,7 @@ int main(int argc, const char **argv) {
 
   auto Files = llvm::makeIntrusiveRefCnt<FileManager>(FileSystemOptions(), OFS);
 
-  auto Driver = std::make_unique<driver::Driver>(
+  auto Driver = std::make_unique<clang::driver::Driver>(

```

Comment:
Are these clang prefixes required?

Reply:
The file imports both the `clang` and `llvm` namespaces. So now that I have added `llvm::driver`, `driver::Driver` is ambiguous.